### DISCUSSION TITLE
Floor spawn the platinum coin

### DISCUSSION THREAD

**Discussion by Ami-OS** [Pryaxis/TShock#2433] [Q&A]
The floor spawn the platinum coin, a lot of platinum coin ( not dropped by players or monsters ).
Question is this is a hack by player ? or tshock defect ?

[![螢幕擷取畫面 2021-08-26 064218](https://user-images.githubusercontent.com/58389183/130874044-4cbc0b32-1fae-49f2-b5ae-f689ecb9af78.png)](https://ami-os.github.io/issues-storage/Pryaxis_TShock_discussions_2433/out1/player/)

server online players: 4

# Edit

- TShock 4.5.5 Upgrade from 4.5.4 ( Both versions have this problem )
- Yes, I have used the plugins. There is also this problem without plugins.
- I recorded a [video](https://ami-os.github.io/issues-storage/Pryaxis_TShock_discussions_2433/out1/player/) and this is related [log](https://github.com/Ami-OS/issues-storage/blob/main/Pryaxis_TShock_discussions_2433/2021-08-24_18-03-00.log).
- When the player walks on the map, the platinum coin will appear on the nearby floor. A lot and only platinum coin.

# extra info

Deployment environment : `Ubuntu Server 20.04.3 LTS`
Mono version : `mono-complete/stable-focal,now 6.12.0.122-0xamarin1+ubuntu2004b1 amd64 [installed]`
Startup parameters :
```bash
mono-sgen "${SCRIPT_DIR}/TerrariaServer.exe" -worldselectpath './world' \
	-world "${worldPath}" \
	-port "${runPort}" \
	-maxplayers 20 \
	-killinactivesocket \
	-forceupdate \
	--stats-optout
```

*If need more information, I'm happy to provide.*

**Comment by csmir** [comment]
This is not nearly enough information to answer the question:

- What TShock version are you currently hosting with?
- Are you playing with other plugins; If so, can you reproduce this on a fresh installment, without other plugins?
- Can you show any screenshots of the source (+console if applicable)?
- Does it look like any of the coins are spawned through duping instead of directly spawning on the ground?

**Reply by Ami-OS** [reply]
- TShock 4.5.5 Upgrade from 4.5.4 ( Both versions have this problem )
- Yes, I have used the plugins. There is also this problem without plugins.
- I recorded a [video](https://ami-os.github.io/issues-storage/Pryaxis_TShock_discussions_2433/out1/player/) and this is related [log](https://github.com/Ami-OS/issues-storage/blob/main/Pryaxis_TShock_discussions_2433/2021-08-24_18-03-00.log).
- When the player walks on the map, the platinum coin will appear on the nearby floor. A lot and only platinum coin.

# extra info

Deployment environment : `Ubuntu Server 20.04.3 LTS`
Mono version : `mono-complete/stable-focal,now 6.12.0.122-0xamarin1+ubuntu2004b1 amd64 [installed]`
Startup parameters :
```bash
mono-sgen "${SCRIPT_DIR}/TerrariaServer.exe" -worldselectpath './world' \
	-world "${worldPath}" \
	-port "${runPort}" \
	-maxplayers 20 \
	-killinactivesocket \
	-forceupdate \
	--stats-optout
```

*If need more information, I'm happy to provide.*

**Reply by Ami-OS** [reply]
What do you think ?
